### PR TITLE
Moar splashes

### PR DESCRIPTION
**Pull request by mobilegmYT**

### CODE REVIEW

**Comment by mobilegmYT** [comment]
@Alvarito050506

File: src/include/splashes.h [hunk 1]
```diff
@@ -54,12 +54,22 @@ char* splashes[] = {
...
 	"FREE!",
 	"RIP MCPIL-Legacy, 2020-2020",
+	"StevePi = popbob confirmed?",
+	"Watch out for StevePi!",
+	"Didn't remove Herobrine!",
+	"Minceraft",
+	"MCPI-Docker",
+	"MCPI-Devs",
+	"RIP PiCraft",
+	"Check out PBPT Anarchy!",
```
Comment (StealthHydra179) [inline src/include/splashes.h:70]: Please remove this advertisement

**Review by StealthHydra179 — CHANGES_REQUESTED** [review]
1 change please

File: src/include/splashes.h [hunk 2]
```diff
@@ -68,7 +68,6 @@ char* splashes[] = {
 	"MCPI-Devs",
 	"RIP PiCraft",
 	"Check out PBPT Anarchy!",
```
Comment (StealthHydra179) [inline src/include/splashes.h:70]: What about `Check out PBPT Anarchy!` ?

Comment (Alvarito050506) [reply src/include/splashes.h:70]: Hm, it's MCPI related...

Comment (StealthHydra179) [reply src/include/splashes.h:70]: Still self advertising, if it was allowed, every single server would be added

Comment (mobilegmYT) [reply src/include/splashes.h:70]: > What about `Check out PBPT Anarchy!` ?

It's a featured server, It'll be featured in mcpil anyways soon I believe.

**Review by StealthHydra179 — APPROVED** [review]

File: src/include/splashes.h (hunk 2 above)
Comment (Alvarito050506) [reply src/include/splashes.h:70]: He is right... We will not accept more splashes from now on anyways.

**Comment by StealthHydra179** [comment]
@Alvarito050506 please review

File: src/include/splashes.h (hunk 2 above)
Comment (mobilegmYT) [reply src/include/splashes.h:70]: > Still self advertising, if it was allowed, every single server would be added

Note that it is one of the only online public mcpi servers except TBR.

**Comment by Alvarito050506** [comment]
> @Alvarito050506 please review

Whoops, just merged

**Comment by StealthHydra179** [comment]
> > @Alvarito050506 please review
> 
> Whoops, just merged

nice

**Pull request merged**